back/app/services/export/xlsx/inputs_generator.rb

```diff
@@ -50,6 +56,14 @@ def generate_for_timeline_project(workbook, project)
 
       def create_phase_sheet(workbook, phase)
         inputs = eager_load_inputs(phase.ideas.submitted_or_published)
+        generate_sheet(workbook, inputs, phase)
+      end
+
+      def create_input_sheet(workbook, input)
```

Comment:
Not really your doing, but we have a bit of a mix here between `generate` and `create` might be nicer to have them all named the same - probably `generate`